pyproject.toml

```diff
@@ -38,7 +38,7 @@ dependencies = [
     'copulas>=0.12.1',
     'ctgan>=0.11.0',
     'deepecho>=0.7.0',
-    'rdt>=1.14.0',
+    'rdt @ git+https://github.com/sdv-dev/RDT.git@rounding-dtype-object',
```

Comment:
can we bump this to use the release candidate [1.15.1.dev0](https://pypi.org/project/rdt/1.15.1.dev0/)? If it works we can go through with the release